CloudyCider/models/EC2PageListener.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  EC2PageListener.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import AWSSDKSwiftCore
+import EC2
+import Foundation
+
+final class EC2PageListener {
+  func loadPage(store: EC2Store) {
```

Comment:
Since `EC2PageListener` has no other properties or state, and `loadPage` takes a single argument of type `EC2Store`, it seems more fitting for `loadPage` to be a function on `EC2Store` instead of the empty `EC2PageListener` class.